packages/dashboard/src/components/appbar.tsx

```diff
@@ -475,17 +464,11 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
                   key={alert.id}
                   title={
                     <React.Fragment>
-                      <Typography variant="caption" display="block" noWrap>
-                        Alert
-                      </Typography>
-                      <Typography variant="caption" display="block" noWrap>
-                        ID: {alert.original_id}
-                      </Typography>
-                      <Typography variant="caption" display="block" noWrap>
-                        Type: {alert.category.toUpperCase()}
-                      </Typography>
-                      <Typography variant="caption" display="block" noWrap>
-                        Created: {new Date(alert.unix_millis_created_time).toLocaleString()}
+                      <Typography>Alert</Typography>
+                      <Typography>ID: {alert.id}</Typography>
+                      <Typography>Title: {alert.title}</Typography>
+                      <Typography>
+                        Created: {new Date(alert.unix_millis_alert_time).toLocaleString()}
```

Comment:
iiuc, there is a list of alerts dialog, and then and there is a alert detail dialog, the alert list dialogs opens the alert detail dialog, having multiple layers of dialog is usually bad UI/UX, but I can see changing this now would disrupt user workflow, so just put a FIXME I guess.

Reply:
This is just a tooltip when the user hovers over one of the alerts listed in the dropdown
![image](https://github.com/open-rmf/rmf-web/assets/5383623/166f7a27-b7b9-4804-83e4-e903fdb09449)